ants.go

```diff
@@ -33,7 +33,7 @@ const (
 	DEFAULT_ANTS_POOL_SIZE = math.MaxInt32
 
 	// DEFAULT_CLEAN_INTERVAL_TIME is the interval time to clean up goroutines.
-	DEFAULT_CLEAN_INTERVAL_TIME = 1
+	DEFAULT_CLEAN_INTERVAL_TIME = 60
```

Comment:
I don't think setting this variable to a oversize value is good idea, threre are different scenarios which should have different interval duration to clean up goroutines, so could you please make `DEFAULT_CLEAN_INTERVAL_TIME ` configurable? May be adding a new function that accepts this variable as its arguments.

Reply:
I will think about it

Reply:
When the pool size is too large , Performance  will drop by the lock . Locking at high concurrency is like an atomic bomb. So I think it should be changed. Because it's the default pool value. Many user will use self-defined pool by NewPool. Of course, it is ok to not change.

Reply:
I think we can modify the New method and periodicallyPurge, it can be used as the next enhancement. I restored it for this pr. May it can be another pr for this.